help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,208 @@
+
+## Prerequisites {#prerequisites}
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
+
+Before activating data to the Outreach destination, you must have a [schema](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+### Prerequisites in Outreach {#prerequisites-destination}
+
+Note the following prerequisites in Outreach, in order to export data from Platform to your Outreach account:
+
+#### You need to have an Outreach account {#prerequisites-account}
+
+Go to the Outreach [sign in](https://accounts.outreach.io/users/sign_in) page to register and create an account, if you do not have one already. Also see support [page](https://support.outreach.io/hc/en-us/articles/207238607-Claim-Your-Outreach-Account).
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](https://experienceleague.adobe.com/docs/experience-platform/xdm/field-groups/profile/segmentation.html?lang=en) if you need guidance on segment statuses.
+
+#### Gather Outreach credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the Outreach CRM destination:
+
+| Credential | Description |
+| --- | --- | --- |
+| Email | Your Outreach account email |
+| Password | Your Outreach account password |
+
+#### Setup custom field labels
+
+Outreach supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the label's to their corresponding segment names instead of keeping the defaults. For example as below:
+
+Outreach settings page for prospects displaying custom fields.
+![Outreach custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+Outreach settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![Outreach custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
+
+> [!NOTE]
+>
+> When updating prospects the label names are not used they are for ease of identification only.
```

Comment:
Please rephrase this note. Something like "Label names are not used when updating prospects. They are only used for ease of identification.".